src/Components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+const Navbar = () => (
+  <StyledHeader>
+    <Link to="/bookshelf" style={{ textDecoration: "none" }}>
+      <HeaderLink>Bookshelf</HeaderLink>
+    </Link>
+    <Link to="/archive" style={{ textDecoration: "none" }}>
+      <HeaderLink>Archive</HeaderLink>
+    </Link>
+    <Link to="/addbook" style={{ textDecoration: "none" }}>
+      <HeaderLink>Add book</HeaderLink>
+    </Link>
+  </StyledHeader>
+);
+
+export default Navbar;
+
+const StyledHeader = styled.ul`
```

Comment:
For code readability I always prefer to declare any function/object that I'm using before actually using it.

Reply:
I moved all styled components to another file, called <componentName>.styled.js and import it.